tests/scenario_account_statement.rst

```diff
@@ -194,18 +194,10 @@ Paid 50 to supplier::
 
 Validate statement::
 
-    >>> statement.click('validate_statement')
+    >>> statement.click('dummy_validate_method')
     >>> statement.state
     'validated'
 
-Try posting a move::
-
-    >>> statement_line = statement.lines[0]
-    >>> statement_line.move.click('post')  # doctest: +IGNORE_EXCEPTION_DETAIL
-    Traceback (most recent call last):
-        ...
-    trytond.modules.account.exceptions.PostError: ...
```

Comment:
on peut conserver ce test non ?

Reply:
@jmousset  je ne trouve pas intérêt de garder ce test car il teste le post sur le move sur les lignes du statement . Mais, nous on ne fait plus ça lors de l'étape de validation on génére plus de move sur les lignes de statement.

Reply:
On ne crée plus de mouvement dans le validate donc on a pas de mouvement a émettre si on ne post pas le relevé cf le commentaire que j'avais mis avant ce test dans le commit "do not delete validate step"

Reply:
@justine-senellart merci !